apps/frontend/src/components/listingCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { Grid, Button, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+
+interface ListingObject {
+  listingID: string;
+  sellerID: string;
+  sellerName: string;
+  title: string;
+  description: string;
+  price: number;
+  dateCreated: string;
+  imageUrl: string;
+}
+
+interface ListingCardProps {
+  listing: ListingObject;
+}
+
+const priceFormatter = new Intl.NumberFormat("en-CA", {
+  style: "currency",
+  currency: "CAD",
+});
+
+const convertDate = (dateString: string) => {
+  const dateTimeObject = new Date(dateString);
+  return dateTimeObject.toDateString();
+};
+
+const ListingCard: React.FC<ListingCardProps> = ({ listing }) => {
+  return (
+    <Grid
+      container
+      direction="column"
+      alignItems="flex-start"
+      justifyContent="flex-start"
+      spacing={3}
+      maxWidth={"29vw"}
```

Comment:
I haven't run the code, but I think that having this set at 29vw for width is going to mess up the mobile view. Ideally this would be 100% width and the spacing would be handled in the parent component, but if that's not simple to implement then a % width is still preferred over viewport units

Reply:
I found this worked best when resizing to ensure that all the cards took up the same amount of space when resizing the page